### ISSUE TITLE
Add a setting to bundle MathJax as part of the output using a URL or file path

### ISSUE THREAD

**Issue by georgestagg** [coursebuilder-ncl/makecourse#91]
Probably a requirement before we can think about #31.

**Comment by georgestagg** [comment]
As of commit 684d31d a custom MathJax URL can be set by adding something like the following to `config.yml`:
```
mathjax_url: 'https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js'
```

**Comment by christianp** [comment]
It would be nice to have that as a command-line option, and bundling a copy of MathJax would be good for archival.
